composer.json

```diff
@@ -64,6 +64,7 @@
     },
     "autoload": {
         "psr-4": {
+            "Rector\\AbstractRector\\": "rules/abstract-rector/src",
```

Comment:
We tend to put abstract classes near to their first/most use.
E.g. `AbstractPHPUnitRector` as at `rules/phpunit` package etc. Could you move it to the place it's used?

Reply:
But it is used in 2 places: by `CreateFunctionToAnonymousFunctionRector` (under `Php72`) and `ArrowFunctionToAnonymousFunctionRector` (under `Downgrade`)... that's why I put it on yet another place...

Tell me exactly where to put it, and I'll do it

Reply:
Which of those 2 you consider first?

Reply:
`php72`, since the `downgrade` one applies to PHP 7.4

Reply:
Let's go with `php72` then